assembl/__init__.py

```diff
@@ -36,6 +35,7 @@
 
 def main(global_config, **settings):
     """ Return a Pyramid WSGI application. """
+    settings = dict(global_config, **settings)
     settings['config_uri'] = global_config['__file__']
```

Comment:
No need to set config_uri key as it is already set in previous line with all other keys from global_config.